easyaudit/models.py

```diff
@@ -1,6 +1,7 @@
 from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I think django is deprecating this in favor of `gettext_lazy`. Can you look into it before merge?

Reply:
Yes you're right `ugettext_lazy` is deprecated and is alias for `gettext_lazy`, commited changes.